src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/induction/InPrisonInterestsEntityMapperTest.kt

```diff
@@ -23,13 +23,13 @@ import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.dto
 class InPrisonInterestsEntityMapperTest {
 
   @InjectMocks
-  private lateinit var mapper: InPrisonInterestsEntityMapperImpl
+  private val mapper = InPrisonInterestsEntityMapperImpl()
```

Comment:
Minor; but did you need to make this change, or was it a hang over from some earlier dev ?

Reply:
Oops, exactly that - a hang over from some earlier dev. I'll update.